### PR TITLE
Create text.py

### PR DESCRIPTION
**Pull request by Abhiigi**
try_1

### CODE REVIEW

File: text.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class student:
+    me='Abhimanyu'
+    def __init__(self,name,regd,mark):
+        self.name=name
+        self.regd=regd
+        self.mark=mark
+        self.lap=self.laptop()
+    def show(self):
```
Comment (ben-india) [inline text.py:8]: Please give a relevant function name.

**Review by ben-india — CHANGES_REQUESTED** [review]

File: text.py (hunk 1 above)
Comment (Abhiigi) [reply text.py:8]: i have made the changes

**Comment by Abhiigi** [comment]
..

**Review by ben-india — APPROVED** [review]

**Review by ben-india — APPROVED** [review]
LGTM

**Pull request merged**